draft-ietf-rats-architecture.md

```diff
@@ -723,7 +723,8 @@ One variation of the background-check model is where the Relying Party
 and the Verifier are on the same machine, performing both functions together. 
 In this case, there is no need for a protocol between the two.
 
-It is also worth pointing out that the choice of model is generally up to the Relying Party. 
+It is also worth pointing out that the choice of model depends on the use case, and that different applications may use different models while still interacting with the same Attester or Verifier.
```

Comment:
I disagree with this wording, which implies that the term "application" inherently implies a Relying Party and not an Attester.
```suggestion
It is also worth pointing out that the choice of model depends on the use case, and that different Relying Parties may use different models while still interacting with the same Attester or Verifier.
```

Reply:
Actually I'm not sure the updated statement is actually true.   If you have a Relying Party that expects Evidence because the Relying Party is using the background check model, then it won't work with an Attester that only sends it Attestation Results and not the Evidence (or claims from the evidence) since the RP's Verifier won't receive the claims it needs to appraise.

I think the statement is more correct in the original and this change should be rejected.